Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: all image test
+
+IMAGE_NAME=codeclimate/codeclimate-duplication
```

Comment:
I *think* the prevailing style is spaces around `=`. Also, how about letting it be env-overridden?

```
IMAGE_NAME ?= codeclimate/codeclimate-duplication
```